src/mock-doc/test/event.spec.ts

```diff
@@ -313,13 +311,11 @@ describe('event', () => {
 
       const composedPath = event.composedPath();
 
-      expect(composedPath).toHaveLength(6);
+      expect(composedPath).toHaveLength(4);
       expect(composedPath[0]).toBe(pElm);
       expect(composedPath[1]).toBe(divElm);
       expect(composedPath[2]).toBe(doc.body);
       expect(composedPath[3]).toBe(doc.documentElement);
-      expect(composedPath[4]).toBe(doc);
-      expect(composedPath[5]).toBe(win);
```

Comment:
I think we still need `composedPath`'s return value to propagate all the way up to `Window`, per https://developer.mozilla.org/en-US/docs/Web/API/Event/composedPath. Is that not the case?

Reply:
I think you're right. I recreated the test "returns the correct path for a simple div element" in the browser and that generated a path of `div`, `body`, `html`, `document`, `window`